lib/arc/storage/local.ex

```diff
@@ -36,4 +36,11 @@ defmodule Arc.Storage.Local do
       Arc.Definition.Versioning.resolve_file_name(definition, version, file_and_scope)
     ])
   end
+
+  defp build_local_request_path(definition, version, file_and_scope) do
```

Comment:
The `local` in this function name feels redundant. Personally, i would drop it from the name, but its not of huge importance